test/testinput/bufr_gpsipw.yaml

```
@@ -0,0 +1,82 @@
+# (C) Copyright 2022 NOAA/NWS/NCEP/EMC
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+
+observations:
+  - obs space:
+      name: bufr
+      obsdatain: "./testinput/gdas.20240806t00z.gpsipw.tm00.bufr_d.nr"
+
+      exports:
+        variables:
+          # MetaData
+          timestamp:
+            datetime:
+              year: "*/YEAR"
+              month: "*/MNTH"
+              day: "*/DAYS"
+              hour: "*/HOUR"
+              minute: "*/MINU"
+
+          latitude:
+            query: "*/CLATH"
+
+          longitude:
+            query: "*/CLONH"
+
+          stationName:
+            query: "*/STSN"
+
+          heightOfStation:
+            query: "*/GNSSRPSQ/ELEV"
+
+          # ObsValue
+          precipitableWater:
+            query: "*/TPWT"
+
+    ioda:
+      backend: netcdf
+      obsdataout: "./testrun/20240806T00Z_ipw_gnssgb_ncep.nc"
+
+      globals:
+        - name: "platformCommonName"
+          type: string
+          value: "GPSIPW"
+
+        - name: "platformLongDescription"
+          type: string
+          value: "M TYPE 012-003  GPS - Integrated Precipitable Water"
+
+      variables:
+        # MetaData
+        - name: "MetaData/dateTime"
+          source: variables/timestamp
+          longName: "Datetime"
+          units: "seconds since 1970-01-01T00:00:00Z" 
+
+        - name: "MetaData/latitude"
+          source: variables/latitude
+          longName: "Latitude"
+          units: "degree_north"
+          range: [-90, 90]
+
+        - name: "MetaData/longitude"
+          source: variables/longitude
+          longName: "Longitude"
+          units: "degree_east"
+
+        - name: "MetaData/stationName"
+          source: variables/stationName
+          longName: "Station Name"
+
+        - name: "MetaData/heightOfStation"
+          source: variables/heightOfStation
+          longName: "Height of Station"
+          units: "m"
+
+        # ObsValue
+        - name: "ObsValue/precipitableWater"
+          source: variables/precipitableWater
+          longName: "GPS IPW"
+          units: "KG/(METER**2)"
```

@mrixlam this is consistent with the way units are presented in our verification yaml
https://github.com/JCSDA-internal/ioda/blob/ad84060522f6fa7ab9c75d227f1d9e72c8bf7a28/share/ioda/yaml/validation/ObsSpace.yaml#L570
```suggestion
          units: "kg m-2"
```